doc/sphinx/changes.rst

```diff
@@ -726,6 +726,44 @@ Changes in 8.10.0
   fixes `#9512 <https://github.com/coq/coq/issues/9512>`_
   by Vincent Laporte).
 
+Changes in 8.10.1
+~~~~~~~~~~~~~~~~~
+
+A few bug fixes and documentation improvements, in particular:
+
+**Kernel**
+
+- Fix proof of False when using |SProp| (incorrect De Bruijn handling
+  when inferring the relevance mark of a function) (`#10904
+  <https://github.com/coq/coq/pull/10904>`_, by Pierre-Marie Pédrot).
+
+**Tactics**
+
+- Fix an anomaly when unsolved evar in Add Ring
+  (`#10891 <https://github.com/coq/coq/pull/10891>`_,
+  fixes `#9851 <https://github.com/coq/coq/issues/9851>`_,
+  by Gaëtan Gilbert).
+
+**Tactic language**
+
+- Fix Ltac1 regression in binding free names in uconstr
```

Comment:
Given that we haven't yet released Ltac2 as part of Coq (this is 8.11 material), I suggest we don't start using the term "Ltac1" in release notes just yet:

```suggestion
- Fix Ltac regression in binding free names in uconstr
```